### PR TITLE
Feature/cms 1209 mithril template solid style

### PR DESCRIPTION
**Pull request by ornellagiau**
all changes for solid template exept buttons

### CODE REVIEW

**Review by VerenaNetlight — APPROVED** [review]

File: lib/units.scss [hunk 1]
```diff
@@ -10,3 +10,10 @@ $mmf-height-label: 1.1em !default;
 // animation
 $mmf-transition--time: 0.1s;
 $mmf-transition: $mmf-transition--time linear;
+
+// text & line styles
+$mmf-line-height: 140.62%;
+$mmf-letter-spacing: 0.05em;
+$mmf-font-normal: 14px;
```
Comment (sagold) [inline lib/units.scss:17]: Dont use absolute font-sizes. At least try using root-relative values (where 16px are default). e.g `$mmf-font-normal: 14/16 + rem`. This enables font-scaling on a global level (increasing font-size on body)

File: components/checkboxform/checkboxform-solid.scss [hunk 2]
```diff
@@ -0,0 +1,21 @@
+@import "../../lib/colors";
+@import "../../lib/units";
+
+
+@mixin checkboxform-solid(){
+    
+
+        label {
+        font-size: $mmf-font-normal;
```
Comment (sagold) [inline components/checkboxform/checkboxform-solid.scss:9]: Be consistent with indentation - may we can automate this

File: lib/colors.scss [hunk 3]
```diff
@@ -2,13 +2,50 @@
 // material design colors https://www.materialui.co/flatuicolors
 // https://material.io/color/#!/?view.left=0&view.right=1&secondary.color=90CAF9&primary.color=ECEFF1
 
+
+//Brand Colors
+$color-basic: (
```
Comment (sagold) [inline lib/colors.scss:7]: We should consider to replace other color-palettes and variables in the near future

File: lib/with-solid-theme.scss [hunk 4]
```diff
@@ -1,33 +1,51 @@
...
+    @include switch-solid();
     @include textarea-solid();
 
-    &.mmf-label {
-        @include font-bold();
-        font-size: $mmf-text-size--label;
-        color: rgba($mmf-color--dark, $mmf-opacity--primary);
+    .mmf-meta {
+        opacity: 1;
+        font-style: normal;
```
Comment (sagold) [inline lib/with-solid-theme.scss:25]: maybe a special or adjusted include for all those font-properties?

File: lib/units.scss [hunk 5]
```diff
@@ -10,3 +10,10 @@ $mmf-height-label: 1.1em !default;
 // animation
 $mmf-transition--time: 0.1s;
 $mmf-transition: $mmf-transition--time linear;
+
+// text & line styles
+$mmf-line-height: 140.62%;
```
Comment (sagold) [inline lib/units.scss:15]: Are you sure, you want percent in line-heights? em are currently used as default (and should behave the same as %). IMHO, em is mor readable, as it references the font-size at the current location, whereas % is a little less specific. Using `1.4062em` should result in same line-height, but shows us, that the value might be a little too specific.

File: lib/colors.scss (hunk 3 above)
Comment (VerenaNetlight) [reply lib/colors.scss:7]: As discussed we'll do it at a later time. I'll add this as an issue.

**Pull request merged**